docs-src/0.5/en/contributing/index.md

```diff
@@ -31,6 +31,12 @@ GitHub CI is much slower at executing all of these than your PC.
 cargo fmt -- src/**/**.rs
 ```
 
+- You might need to install some packages on Linux (Ubuntu/deb) before the following commands will complete successfully (there is also a Nix flake in the repo root):
```

Comment:
Are these dependancies for multiple different package in the CLI or just dioxus desktop? If it is just for dioxus desktop, it might be useful in the platform specific dependancies section in the getting started guide

Reply:
These dependencies were required simply to run the next line 
`cargo check --workspace --examples --tests`